package.json

```diff
@@ -50,7 +50,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
+    "build": "gatsby build && cp robots.txt public/robots.txt",
```

Comment:
@Kameshwaran a better way to do this is
https://www.gatsbyjs.org/docs/static-folder/

Reply:
good catch. I wonder how i forget this basic stuff. I've already done this for static scripts